friends/src/App.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import axios from 'axios';
+import Friends from './Components/Friends';
+import Friend from './Components/Friend';
+import DisplayFriend from './Components/DisplayFriend';
+import FriendForm from './Components/FriendForm';
+import EditFriend from './Components/EditFriend';
+
+/***************************************************************************************************
+ ********************************************* Variables *******************************************
+ **************************************************************************************************/
+const urlLinks = {
+  home: '/',
+  friend: `/friend`,
+  editFriend: `/friend/edit`
+};
+
+/***************************************************************************************************
+ ********************************************* Component *******************************************
+ **************************************************************************************************/
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => console.log(err));
+  }
+
+  addFriend = newFriend => {
+    axios
+      .post(`http://localhost:5000/friends`, newFriend)
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => console.log(err));
+  };
+
+  deleteFriend = (e, id) => {
+    e.preventDefault();
+    axios
+      .delete(`http://localhost:5000/friends/${id}`)
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => console.log(err));
+  };
+
+  editFriend = (id, editedFriend) => {
+    axios
+      .put(`http://localhost:5000/friends/${id}`, editedFriend)
+      .then(res => {
+        this.setState({ friends: res.data });
+      })
+      .catch(err => console.log(err));
+  };
+
+  render() {
+    return (
+      <div className='App'>
```

Comment:
Good spot for a `<Fragment>` since it doesn't look like you're using the className for any styling.